### ISSUE TITLE
App Store Connect Operation Error

### ISSUE THREAD

**Issue by PomegranateApps** [persona-id/persona-ios-sdk#9]
I'm not able to submit a binary to the App Store with the latest version of Persona (2.6.1). 

This is the error I'm getting from App Store Connect.

_The app references non-public selectors in Payload/MyApp.app/Frameworks/Persona2.framework/Persona2: setInputImage:_

**Comment by jacoblange-dev** [comment]
What version were you using prior? A search for `setInputImage` within our codebase yield no results. 

There is however one usage of https://developer.apple.com/documentation/coreimage/cifalsecolor/3228258-inputimage but nothing about our usage of that looks non-public to me?

```
private func updateColor(image: CIImage, strokeColor: UIColor, fillColor: UIColor) -> CIImage? {
    let colorFilter = CIFilter.falseColor()
    colorFilter.inputImage = image
    colorFilter.color0 = CIColor(color: strokeColor)
    colorFilter.color1 = CIColor(color: fillColor)
    return colorFilter.outputImage
}
```

The above usage of `colorFilter.inputImage = image` first went live in v2.5.1.

Any more info you can provide?

**Comment by PomegranateApps** [comment]
I was using 2.4 prior. This is my first time trying to submit a binary with the latest version.

That's the only error message I get when I try to upload the app to App Store Connect.

**Comment by jacoblange-dev** [comment]
Hmm odd well I ran into this as well today, App Store Connect is rejecting our framework via Xcode but not when using fastlane. Not sure what the issue is, I believe a bug on Apple side of things. Anyhow, I've put in a workaround which changes the call to `colorFilter.inputImage = image` to `colorFilter.setValue(image, forKey: "inputImage")` and App Store Connect is happy with that so I will release a fix in our next patch release (2.7.1).
